index.html

```diff
@@ -0,0 +1,441 @@
+<!DOCTYPE html>
```

Comment:
Love the overall look of this! Looks very clean and professional close to the actual twitter design! Good job Rasheed!